src/vs/workbench/contrib/comments/browser/commentsAccessibleView.ts

```diff
@@ -84,3 +112,41 @@ class CommentsAccessibleContentProvider extends Disposable implements IAccessibl
 		return this.provideContent();
 	}
 }
+
+class CommentsThreadWidgetAccessibleContentProvider extends Disposable implements IAccessibleViewContentProvider {
+	readonly id = AccessibleViewProviderId.CommentThread;
+	readonly verbositySettingKey = AccessibilityVerbositySettingId.Comments;
+	readonly options = { type: AccessibleViewType.View };
+	constructor(@ICommentService private readonly _commentService: ICommentService,
+		@IEditorService private readonly _editorService: IEditorService,
+		@IUriIdentityService private readonly _uriIdentityService: IUriIdentityService,
+	) {
+		super();
+	}
+	// TODO: actions
+	public actions = [];
+	provideContent(): string {
+		if (!this._commentService.activeCommentInfo) {
+			throw new Error('No current comment thread');
+		}
+		const value = this._commentService.activeCommentInfo.comment?.body;
+		return typeof value === 'string' ? value : value?.value ?? '';
+	}
+	onClose(): void {
+		const commentInfo = this._commentService.activeCommentInfo;
+		if (!commentInfo) {
+			return;
+		}
+		// TODO: is there a way to focus the comment not the thread?
```

Comment:
is there a way to do this?

Reply:
We'll need to expand on `CommentThreadFocus`. You can leave this TODO for me and I'll add it after the PR is merged. 

https://github.com/microsoft/vscode/blob/5da1e4ae706213256ab61b7b4fc21a1b8c9a4c19/src/vs/workbench/contrib/comments/browser/commentThreadZoneWidget.ts#L34-L38